.github/workflows/weekly.yml

```
@@ -21,14 +21,14 @@ jobs:
           fetch-depth: 0
       - name: update-xla-commit-hash
         continue-on-error: true
-        uses: ./.github/actions/update-commit-hash
+        uses: pytorch/test-infra/.github/actions/update-commit-hash@main
         with:
           repo-name: xla
           branch: master
```

```suggestion
          branch: master
          pin-folder: .github/ci_commit_pins
          test-infra-ref: main
```

Those params now need to be set on the new GHA.